ERCS/erc-7841.md

```diff
@@ -0,0 +1,545 @@
+---
+eip: 7841
+title: Cross-chain Message Format and Mailbox
+description: A mailbox API and message format to provide a unified cross-chain developer experience
+author: Ellie Davidson (@elliedavidson), Alex Xiong (@alxiong), Philippe Camacho (@philippecamacho), and Ben Fisch (@benafisch)
+discussions-to: https://ethereum-magicians.org/t/erc-7841-cross-chain-message-format-and-mailbox/22185
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-12-12
+---
+
+## Abstract
+
+This ERC proposes a **mailbox API** and **message format** for sending and receiving data between L2s. This standard makes it easier for developers to build cross-chain applications that work over a variety of VMs, chain settlement mechanisms (e.g., ZK or optimistic settlement), and messaging protocols (e.g., synchronous or asynchronous protocols). Example applications include an intent-based bridge, a liquidity-unifying DEX operating across multiple chains, or a cross-chain lending application.
```

Comment:
You abstract should sketch out a high level technical overview so readers get a sense of _how_ your proposal accomplishes its goals.